src/components/events/EventForm.jsx

```diff
@@ -33,7 +35,12 @@ class EventForm extends React.Component {
 
   handleChange = (e) => {
     const { name, value } = e.target;
-    this.setState({ [name]: value });
+    if (name === 'eventDate') {
+      this.setState({ eventDate: value.substring(8,10) + '/' + value.substring(5,7) + '/' + value.substring(0,4)});
+    }
+    else {
+      this.setState({ [name]: value });
```

Comment:
There's a fun way we could get lines 38 - 43 condensed into one line, theoretically.

If i run this `this.setState({ [name]: value })` when name is eventDate, the key of the object will still be valid. So we could actually condense both the setState on lines 39 and 42 into one setState function.

```
this.setState({ [name]: name === 'eventDate' ? value...... : value });
```

Reply:
Once again, you're a genius.